src/helper.js

```diff
@@ -10,7 +12,7 @@ function asCachableGeojson(geojson) {
     geojson.type = geojson.type || 'FeatureCollection'
     geojson.features = geojson.features || [];
     geojson.metadata = geojson.metadata || {};
-    return geojson;
+    return _.cloneDeep(geojson);
```

Comment:
I understand why this is necessary, but definitely am curious about the performance with geojson with a large number of features.

Reply:
sure, same here.  I assume it doubles memory usage.  I guess people should probably use the Redis cache (instead of in-memory) for production deployments.

Reply:
Well, I guess it doubles memory usage, but only during the request lifecycle.  After the request is complete, memory usage would be the same as prior to this change.

Would be interesting to have some eviction loop running.  Once something in the store is expired, eject it from the cache and free up memory.